lib/component.js

```diff
@@ -425,11 +425,11 @@ class Component extends WebComponent {
           }),
         );
       } catch (error) {
-        const message = `Error while rendering ${this.toString()}`;
+        const message = `Error while rendering`;
         this._logError(message, this, `\n`, error);
         this.dispatchEvent(
           new CustomEvent(`renderError`, {
-            detail: {message, error},
+            detail: {message, error, component: this},
```

Comment:
```suggestion
            detail: {error, component: this},
```
The "message" field now yields absolutely no information not already in the event name "renderError" so let's get rid of it